vars/main.yml

```diff
@@ -15,9 +15,9 @@ __bootloader_packages:
   - grubby
 __bootloader_default_grub: /etc/default/grub
 __bootloader_uefi_conf_dir: >-
-  {%- if ansible_distribution == 'RedHat' -%}
-  /boot/efi/EFI/redhat/
-  {%- elif ansible_distribution in ['SLES', 'SLES_SAP'] -%}
+  {%- if ansible_os_family == 'RedHat' -%}
+  /boot/efi/EFI/{{ ansible_distribution | lower }}/
+  {%- elif ansible_os_family == Suse -%}
```

Comment:
are you sure that `SLES_SAP` is in the `Suse` os_family?

Reply:
Yes, see https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/facts/system/distribution.py#L519